server/src/util/my-repos.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import gql from 'graphql-tag';
+import {GitHub} from '../github';
+import {MyReposQuery, MyReposQueryVariables} from '../gql-types';
+
+const MAX_STATS_RETRIES = 5;
+
+// Variables that impact the scoring algorithm.
+const SCORE_THRESHOLD = 20;
```

Comment:
Maybe an extra sentence for SCORE_THRESHOLD and CONTRIBUTION_SIZE?

Reply:
Done.